CONTRIBUTING.md

```diff
@@ -323,3 +323,22 @@ by creating a pre-commit git hook as follows:
     ```bash
     chmod +x ./.git/hooks/prepare-commit-msg
     ```
+
+### Code Directories
+
+Here, we briefly give you an overview of Keptn Lifecycle Toolkit's directories.
+
+```bash
+.
+├── dashboards        - contains dashboards for the KLT.
+├── docs              - docs and tutorials related to KLT
+├── examples          - usecases to demonstrate the capabilities of the KLT.
+├── functions-runtime - runtime definitions for executing KLT tasks
+├── helm              - application health checks introduces by KLT.
+├── klt-cert-manager  - secures webhooks for KLT operator's access to Kubernetes API server with valid certificates.
+├── metrics-operator  - 
+├── operator          - 
+├── scheduler         -
+└── test              -
```

Comment:
contains test artifacts and configuration files used for testing Keptn components and workflows.